sdk/tables/data-tables/MigrationGuide.md

```diff
@@ -0,0 +1,387 @@
+
+### Batch Transactions
+Previously with `azure-storage`, creating and executing a transactional batch operation involved creating a `TableBatch` which contained the TableOperation(s) to be executed. The result from ExecuteBatch was a TableBatchResult which is essentially a collection of TableResults.
+
+```javascript
+const azure = require('azure-storage');
+var tableService = azure.createTableService("<connection-string>");
+
+const tableName = "<table-name>"
+
+const task1 = {
+  PartitionKey: {'_':'hometasks'},
+  RowKey: {'_': '1'},
+  description: {'_':'Take out the trash'},
+  dueDate: {'_':new Date(2015, 6, 20)}
+};
+const task2 = {
+  PartitionKey: {'_':'hometasks'},
+  RowKey: {'_': '2'},
+  description: {'_':'Wash the dishes'},
+  dueDate: {'_':new Date(2015, 6, 20)}
+};
+
+const batch = new azure.TableBatch();
+
+batch.iniertEntity(task1, {echoContent: true})
+batch.iniertEntity(task2, {echoContent: true})
+
+tableService.executeBatch(tableName, batch, function(error, result, response){
+  if(!error) {
+    console.log("Batch completed");
+  }
+});
+```
+
+Now in `@azure/data-tables`, you just need to create an array of operations and pass it to the `submitTransaction` method on the `TableClient`
+
+```javascript
+const { TableClient, AzureNamedKeyCredential } = require('@azure/data-tables');
+const tableName = "<table-name>"
+const tableClient = new TableClient(tablesEndpoint, tableName, new AzureNamedKeyCredential("<accountName>","<accountKey>"));
+
+const task1 = {
+  PartitionKey: "hometasks",
+  RowKey: "1",
+  description: "Take out the trash",
+  dueDate: new Date(2015, 6, 20)
+};
+const task2 = {
+  PartitionKey: "hometasks",
+  RowKey: "2",
+  description: "Wash the dishes",
+  dueDate: new Date(2015, 6, 20)
+};
+
+const tableActions = [["create", task1], ["create", task2]];
+
+
+await tableClient.submitTransaction(tableActions);
+console.log("Transaction completed")
+```
+
+For convenience also provide a helper that works in a similar way as `TableBatch` in `azure-storage`.
+
+```javascript
+const { TableClient, AzureNamedKeyCredential, TableTransaction } = require('@azure/data-tables');
+const tableName = "<table-name>"
+const tableClient = new TableClient(tablesEndpoint, tableName, new AzureNamedKeyCredential("<accountName>","<accountKey>"));
+
+const task1 = {
+  PartitionKey: "hometasks",
+  RowKey: "1",
+  description: "Take out the trash",
+  dueDate: new Date(2015, 6, 20)
+};
+const task2 = {
+  PartitionKey: "hometasks",
+  RowKey: "2",
+  description: "Wash the dishes",
+  dueDate: new Date(2015, 6, 20)
+};
+
+const transaction = new TableTransaction();
+transaction.createEntity(task1);
+transaction.createEntity(task2);
+
+await tableClient.submitTransaction(transaction.actions);
+console.log("Transaction completed")
+```
+
+### Sequential Actions
+
+Previously in `azure-storage`, all the operations took a callback used to execute actions in sequence. For example, to create a table and then insert two entities
```

Comment:
This sentence seems to have ended too early. Did you intend to have more after "insert two entities..."?